client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
Thinking about it.. maybe it *would* be the easiest to just expose the tree.

Basically the gist of what's needed (that should actually work for most cases) is:

```
klass = TreeBuilder.class_for_type(params[:tree_type])
tree = klass.new(params[:tree_name], params[:tree_type], @sb, true)
render :json => { :tree => tree.locals_for_render }
```